.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,90 @@
+    pipelinesascode.tekton.dev/task-15: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/run-tfa-classifire.yaml"
+    pipelinesascode.tekton.dev/task-16: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-results-to-dashboard.yaml"
+    pipelinesascode.tekton.dev/task-17: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-slack-notification.yaml"
+    pipelinesascode.tekton.dev/task-18: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/setup-testing-accounts.yaml"
+    pipelinesascode.tekton.dev/task-19: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/uninstall-cluster.yaml"
+    pipelinesascode.tekton.dev/task-20: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upload-artifacts.yaml"
+    pipelinesascode.tekton.dev/task-21: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-triggers-tests.yaml"
+    pipelinesascode.tekton.dev/task-22: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pipelines-tests.yaml"
+    pipelinesascode.tekton.dev/task-23: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pac-tests.yaml"
+    pipelinesascode.tekton.dev/task-24: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-operator-tests.yaml"
+    pipelinesascode.tekton.dev/task-25: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/whitelist-domains.yaml"
+    pipelinesascode.tekton.dev/max-keep-runs: "2"  
+spec:
+  pipelineRef:
+    name: acceptance-tests
+  params:
+    - name: GIT_RELEASE_TESTS_URI
+      value: "{{ repo_url }}"
+    - name: GIT_RELEASE_TESTS_BRANCH
+      value: "{{ revision }}"
+    - name: CLUSTER_POOL
+      value: "openstack"
+    - name: INSTALLER
+      value: "hive"
+    - name: CLI_TESTS_TYPE
+      value: downstream
+    - name: OPERATOR_ENVIRONMENT
+      value: pre-stage
+    - name: OPERATOR_VERSION
+      value: "5.0"
+    - name: TEST_SUITES
+      value:
+        # - cli-tests-fedora
+        - release-tests-versions
+        - release-tests-chains
+        - release-tests-clustertasks
+        - release-tests-clustertasks-s2i
+        - release-tests-clustertasks-multiarch
+        - release-tests-clustertasks-disconnected
+        - release-tests-metrics
+        - release-tests-pipelines
+        - release-tests-results
+        - release-tests-triggers
+        - release-tests-triggers-tls
+        # - tutorial-tests
+        - triggers-upstream-tests
+        # - pipelines-upstream-tests
+        # - operator-upstream-tests
+        - release-tests-rbac
+        - release-tests-auto-prune
+        - release-tests-addon
+    - name: TKN_DOWNLOAD_URL
+      value: https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.13.0/tkn-linux-amd64.tar.gz
```

Comment:
either update it to 1.15.0 or remove this param, CLI_TESTS_TYPE and cli-tests-fedora from TEST_SUITES